concepts/char/about.md

```diff
@@ -49,23 +49,23 @@ Thereby Chars can only be concatenated with Strings.
 
 ## Properties methods
 
-Crystal has a few methods that can be used to get information about a Char, like if it is a number or a letter, or if it is uppercase or lowercase.
-Here is a list of some of the methods, for a full list see the [Char API][char-api]:
+Crystal has a few methods for getting information about a Character, such as whether it is a number or a letter or whether it is uppercase or lowercase.
+Here is a list of some of the methods, for a complete list see the [Char API][char-api]:
 
-| Method                           | Description                                          |
+| Method                           | Description                                          |
```

Comment:
I've seen this in a few of these PRs -- do you know what's actually different? At a glance it appears that the before and after are both spaces.

Reply:
I think my editor might change something, I am not sure why.